subsystems/intakeSubsystem.py

```diff
@@ -0,0 +1,43 @@
+import wpilib
+import commands2
+import phoenix5
```

Comment:
I don't think you need to import phoenix5 anymore, since we aren't using any TalonFX motor controllers, right?